cost/azure/instances_log_analitics_utilization/README.md

```diff
@@ -39,6 +39,17 @@ This policy has the following input parameters required when launching the polic
 - *Action Tag Key:Value* - The tag key:value pair to set on an instance that is underutilized.
 - *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
 
+### Required RightScale Roles
+
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```

Comment:
for consistency please change to Cloud Management - credential_viewer

Reply:
Changed to Cloud Management - credential_viewer